### DISCUSSION TITLE
CKAN for Windos

### DISCUSSION THREAD

**Discussion by Oswaldoivann** [ckan/ckan#7484] [General]
"Hello, I'm new here. My first request is to install CKAN on Windows. I've read the documentation for version 2.10, but the first option provided is for Ubuntu. Is it possible to install CKAN on Windows? If so, is there any documentation or website available that provides the procedure for doing so?"

**Comment by pdelboca** [comment]
Hello @Oswaldoivann !

We do not support it. While we receive occasional inquiries and there may be individuals who might try to use it, we have no plans to add Windows support now or in the future.
